community/command-line/src/main/java/org/neo4j/commandline/admin/AdminCommandSegment.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.commandline.admin;
+
+public abstract class AdminCommandSegment
```

Comment:
First naming: `AdminCommandSegment` -> `AdminCommandCategory`.

Second:
My first thought here was that it should have been an `interface` and not an `abstract class`, however it has single method equivalent to a `toString` method. So now I'm wondering why it's a class at all when it could just be a String.